vaccine_feed_ingest/runners/tx/arcgis/normalize.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("tx/arcgis/normalize.py")
+
+VACCINES_FIELD = {
+    "JJ_AVAILABLE": schema.Vaccine(vaccine="janssen"),
+    "JJ_AVAILABLE2": schema.Vaccine(vaccine="janssen"),
+    "MODERNA_AVAILABLE": schema.Vaccine(vaccine="moderna"),
+    "MODERNA_AVAILABLE2": schema.Vaccine(vaccine="moderna"),
+    "PFIZER_AVAILABLE": schema.Vaccine(vaccine="pfizer"),
+    "PFIZER_AVAILABLE2": schema.Vaccine(vaccine="pfizer"),
```

Comment:
Thank you for normalizing these! Last night we updated [the expected values](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema#vaccine-type-enum) for compatibility with another system. Could you update these as well?

Reply:
Done!